docs/discover/get-started-discover.asciidoc

```diff
@@ -167,6 +167,18 @@ your data appears in a map.
 [role="screenshot"]
 image:images/discover-maps.png[Map containing documents]
 
+[float]
+[[in-table-search]]
+==== Find and highlight values in the table
+
+Use the in-table search to find and highlight specific values beyond what's currently visible on your screen. 
+
+The in-table search looks for all matching values in all results and pages currently loaded in the table. The number of results loaded depends on the <<document-explorer-sample-size,Sample size>> option value. If you load more results, the search automatically updates and reflects the new number of matching values, if any more are found.
+
+TIP: You can navigate between results with your keyboard by pressing "Enter" to go to the next result, and "Shift + Enter" to go to the previous result.
+
+image::https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt30bf5f8b9a45ab74/67c234a787966d9fbc994ce0/in-table-search-demo.gif[Using the in-table search and navigating through the matches]
+
```

Comment:
Same as in https://github.com/elastic/docs-content/pull/807#discussion_r2000633675